iFixit/react-commerce
# Tests: Remove Cypress

## Summary
Removed all of the Cypress-related files including:
- test-helpers
- cypress and cypress-related packages/libraries. Used `pnpm remove -D <package-name>`.
- all cypress tests and configs
- cypress workflow

## QA notes
1. Go to the collections page (either `/Parts` or `/Tools`) and click the pagination button for the next search result. 
Then make sure it scrolls smoothly (same as on master).
2. Run `git grep -i cypress` and make sure no actual code shows up in the result. 
Note: there should be only one cypress comment in `playwright/utils.ts`. 

closes: #1239 #1111
